lib/etc/openturns.conf.in

```diff
@@ -800,8 +800,13 @@
   <LeastSquaresExpansion-DecompositionMethod       value_str="QR" />
 
   <!-- OT::LinearModelAlgorithm parameters -->
-  <LinearModelAnalysis-Identifiers                 value_int="3"      />
   <LinearModelAlgorithm-DecompositionMethod        value_str="QR"      />
+  
+  <!-- OT::LinearModelAnalysis parameters -->
+  <LinearModelAnalysis-Identifiers            value_int="3"      />
```

Comment:
already defined in #2330

Reply:
That is true: it will create a conflict when the PR is merged. I will handle this.